.github/workflows/ci.yml

```diff
@@ -70,14 +70,14 @@ jobs:
                       packages/*/node_modules/
                       plugins/*/node_modules/
                       libraries/*/node_modules/
-                  key: node_modules-${{hashFiles('yarn.lock','patches/*.patch')}}
+                  key: node_modules-${{hashFiles('yarn.lock','patches/*.patch', 'packages/*/package.json', 'plugins/*/package.json')}}
```

Comment:
This is required because dependencies on internal projects don't change the yarn lockfile, but they do need yarn to run to set up symlinks